wikipedia-sentences/app.py

```diff
@@ -79,6 +98,9 @@ def main(task, num_docs, top_k):
     config()
     workspace = os.environ['JINA_WORKSPACE']
 
+    shutil.rmtree('workspace', ignore_errors=True)
+    os.makedirs('workspace')
+    '''
```

Comment:
Why are we commenting this?

Reply:
Because in the current version there is no need to pass in task arguments like "index" and "query" as index and query are done in one flow without workspace

Reply:
Then shouldn't we remove it?